xcdat/spatial.py

```diff
@@ -309,25 +309,32 @@ def _validate_axis_arg(self, axis: List[SpatialAxis]):
             # Check the axis coordinate variable exists in the Dataset.
             get_axis_coord(self._dataset, key)
 
-    def _validate_domain_bounds(self, domain_bounds: xr.DataArray):
-        """Validates the ``domain_bounds`` arg based on a set of criteria.
+    def _force_domain_order_low_to_high(self, domain_bounds: xr.DataArray):
+        """Reorders the ``domain_bounds`` low-to-high.
+
+        This method ensures all lower bound values are less than the upper bound
+        values (``domain_bounds[:, 1] < domain_bounds[:, 1]``).
 
         Parameters
         ----------
         domain_bounds: xr.DataArray
-            The bounds of an axis
+            The bounds of an axis.
 
-        Raises
+        Returns
         ------
-        TypeError
-            If the ``domain_bounds`` of a grid cell are not ordered low-to-high.
+        xr.DataArray
+            The bounds of an axis (re-ordered if applicable).
         """
         index_bad_cells = np.where(domain_bounds[:, 1] - domain_bounds[:, 0] < 0)[0]
+
         if len(index_bad_cells) > 0:
-            raise ValueError(
-                "The bounds have unexpected ordering. A lower bound has a "
-                "greater value than an upper bound."
-            )
+            new_domain_bounds = domain_bounds.copy()
+            new_domain_bounds[index_bad_cells, 0] = domain_bounds[index_bad_cells, 1]
+            new_domain_bounds[index_bad_cells, 1] = domain_bounds[index_bad_cells, 0]
+
+            return new_domain_bounds
+        else:
+            return domain_bounds
```

Comment:
```suggestion

        return domain_bounds
```

Just saw that this can be simplified further (nitpick)